uint16    controller_id
uint16    n_actuated_joints
float64[] joint_angles
float64[] joint_velocities